src/SourceBuild/content/eng/extract-sdk-archive.proj

```diff
@@ -8,13 +8,20 @@
           BeforeTargets="AfterBuild"
           DependsOnTargets="DetermineSourceBuiltSdkVersion"
           Inputs="$(SdkTarballPath)"
-          Outputs="$(DotNetSdkExtractDir)">
+          Outputs="$(BaseIntermediateOutputPath)ExtractSdkArchive.complete">
     <MakeDir Directories="$(DotNetSdkExtractDir)" />
+
     <Exec Condition="'$(ArchiveExtension)' == '.tar.gz'"
           Command="tar -xzf $(SdkTarballPath) -C $(DotNetSdkExtractDir)" />
     <Unzip Condition="'$(ArchiveExtension)' == '.zip'"
            SourceFiles="$(SdkTarballPath)"
            DestinationFolder="$(DotNetSdkExtractDir)" />
+
+    <Message Text="Extracted SDK archive file to '$(DotNetSdkExtractDir)'." Importance="high"  />
```

Comment:
```suggestion
    <Message Text="Extracted SDK archive file '$(SdkTarballPath)' to '$(DotNetSdkExtractDir)'." Importance="high" />
```

Reply:
If you find that additional information in the message useful, sure.